hnn_core/cell.py

```diff
@@ -112,6 +112,7 @@ def __init__(self, soma_props, gid=None):
         self.rec_v = h.Vector()
         self.rec_i = dict()
         self._gid = None
+        self.tonic_biases = []
```

Comment:
nitpick, but in general you should prefer instantiating with class name, e.g., `list()`. We've had cases in the past where people misread `[]` for `{}` and vice-versa

Reply:
replaced in all the files I've touched in this PR